content/general/concepts/hexadecimal/hexadecimal.md

```diff
@@ -0,0 +1,47 @@
+---
+Title: 'Hexadecimal'
+Description: 'Hexadecimal(hex, hex numbers, or base-16) is a numeral system that uses a base of 16 as opposed to the standard decimal system, base 10.'
+Subjects: 
+    - 'Web Development'
+    - 'Web Design'
+    - 'Computer Engeneering'
+Tags:
+    - 'colors'
+CatalogContent:
+    - 'paths/computer-science'
+    - 'paths/code-foundations'
+---
+
+**Hexadecimal** (hex, hex numbers, or base-16) is a numeral system that uses a base of 16 as opposed to the standard decimal system, base 10. The two have the first ten digits in common, 0-9. The remaining digits 10-15 are represented by the letters A-F in base 16. The letters are case-insensitive. 
+
+| Decimal   | Hex       |
+|---------- |-----------|
+|     1     |     1     |    
+|     2     |     2     |
+|     3     |     3     |
+|     4     |     4     |
+|     5     |     5     |
+|     6     |     6     |
+|     7     |     7     |
+|     8     |     8     |
+|     9     |     9     |
+|    10     |     A     |
+|    11     |     B     |
+|    12     |     C     |
+|    13     |     D     |
+|    14     |     E     |
+|    15     |     F     |
+
+## Power Of 2
+
+Hexadecimal, as well as binary(base 2) and octal(base 8) number systems, have bases that are powers of 2. This makes it easy to convert between these systems. 
+
+
+## Syntax
+
+A hexadecimal number is prefixed with `0x`.
+
+
+## Hex Colors
+
+Hex numbers often represent colors(hex colors) and are specified with `#RRGGBB`(red, green, and blue). One example of a hex color is `#0000FF` and represents the color blue. We know this because red and green is set to their lowest value, `00`, and blue set to their highest value, `FF`. A hex color is prefixed with `#`.
```

Comment:
```suggestion
Hex numbers are often used to represent colors (hex colors) and are specified in the following way: `#RRGGBB`(red, green, and blue). One example of a hex color is `#0000FF` which represents the color blue. This is because red and green are set to their lowest value, `00`, and blue is set to its highest value, `FF`.
```